packages/components/addon/components/hds/stepper/step/indicator.hbs

```diff
@@ -2,7 +2,7 @@
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: MPL-2.0
 }}
-<div class={{this.classNames}} ...attributes aria-label={{@status}}>
```

Comment:
Is the status information valuable? Should it be moved to somewhere else?

Reply:
By removing the incorrect `aria-label` we will at least stop their tests from failing. 

I'm looking at the component to see if there's a better way to do this but IIRC this was supposed to be integrated into the consumer's components and not used on its own; I'll need to do a little more checking to see what the right next fix is. 👍